NitroxClient/Debuggers/NetworkDebugger.cs

```diff
@@ -208,6 +219,17 @@ private void AddPacket(Packet packet, bool isSent)
             }
         }
 
+        private static string BytesToString(uint byteCount)
+        {
+            string[] suf = { "B", "KB", "MB", "GB", "TB" };
+            if (byteCount == 0)
+                return "0" + suf[0];
+            long bytes = Math.Abs(byteCount);
+            int place = Convert.ToInt32(Math.Floor(Math.Log(bytes, 1024)));
+            double num = Math.Round(bytes / Math.Pow(1024, place), 1);
+            return (Math.Sign(byteCount) * num).ToString() + suf[place];
+        }
+
```

Comment:
Recommend putting this in `NitroxModel.Extensions.cs` as an extension method for `uint`.
```suggestion
        public static string AsByteUnitText(this uint byteSize)
        {
            // Uint can't go past 4GB, so we don't need to worry about overflow.
            string[] suf = { "B", "KB", "MB", "GB" };
            if (byteSize == 0)
            {
                return $"0{suf[0]}";
            }
            int place = Convert.ToInt32(Math.Floor(Math.Log(byteSize, 1024)));
            double num = Math.Round(byteSize / Math.Pow(1024, place), 1);
            return num + suf[place];
        }
```